src/app/providers.tsx

```diff
@@ -0,0 +1,12 @@
+'use client'
```

Comment:
Copied & Pasted from the following official article.
https://chakra-ui.com/getting-started/nextjs-guide